modules/ROOT/pages/publish-content.adoc

```diff
@@ -6,6 +6,16 @@ endif::[]
 
 With ACM you can create, organize, and publish content articles including news, tutorials, developer blog posts, and events.
 
+You can author ACM content in either of the following ways:
+
+. In https://help.salesforce.com/articleView?id=community_managed_content_overview.htm&type=5[SalesforceCMS].
+. In https://help.salesforce.com/articleView?id=communities_cms_connect.htm&type=5[an external CMS system through 
+CMS Connect].
+
+To display content in community pages, you must organize content in COLLECTIONS using the *Manage CMS content* section 
+of the ACM admin interface, as shown in https://drive.google.com/open?id=1ZxQ2lvivxnXDKB8qfJX3qZM-Szrzx4Dl[this 
+video with a similar talk track].
+
 == Create a New Article
 
 In the ACM administrator interface, select *Manage CMS Content*.
```

Comment:
we should describe here the happy path to create an article in salesforce CMS:
- in ACM admin, select manage CMS content
- click on Salesforce CMS to go to Salesforce CMS component
- Open the workspace for the community you're working with
- see list of existing articles - click new to create a new one
- enter title, excerpt, image, and body
- click save draft and keep editing or click PUBLISH to make it live (if it is in a published collection as explained below on the collections section)